docs/source/fundamentals/parallelism.md

```diff
@@ -0,0 +1,11 @@
+# Parallelism
+
+Streaming supports data parallelism as well as sequence/tensor parallelism.
+
+- **Data Parallelism**: Streaming supports this by default. Each device will get a unique portion of
+each global batch. Samples are not replicated across devices. FSDP and HSDP both fall under this
```

Comment:
Can we also add `DDP` along with `FSDP` and `HSDP`?

Reply:
yep, added